README.md

```diff
@@ -1 +1,96 @@
-# presto-alibaba-maxcompute-connector
+# MaxCompute Connector
+
+The MaxCompute connector enables direct querying and processing of data stored in the MaxCompute data warehouse. This connector is particularly useful for implementing data
+integration and analysis between MaxCompute and other systems, such as Hive.
+
+## MaxCompute Connector Setup
+
+### Prerequisites
+
+Before using the MaxCompute connector, please ensure the following conditions are met:
+
+- Access permissions: Ensure you have valid access rights to a MaxCompute project and that you have obtained the necessary authentication credentials.
+- Network setup: For stable data transmission, the use of Aliyun VPC network is recommended. Check out
+  the [MaxCompute Network Configuration Guide](https://help.aliyun.com/zh/maxcompute/user-guide/network-connection-process) for more information.
+- Resource group configuration: Possession of an exclusive Tunnel resource group is required (although MaxCompute currently allows access to shared resource groups, this could
+  change at any time). [Exclusive resource group](https://help.aliyun.com/zh/maxcompute/user-guide/purchase-and-use-exclusive-resource-groups-for-dts).
+- (Optional) Schema activation: Enable Schema mode to use different directories in Presto. Refer
+  to [How to Enable Schema Model in MaxCompute](https://help.aliyun.com/zh/maxcompute/user-guide/schema-related-operations) for more information.
+
+### Obtain the Connector JAR
+
+To use the MaxCompute connector, you first need to obtain the JAR file by building the repository. Use the Maven command `mvn clean package` to build the project and generate the
+JAR. Once built, place the JAR file into the `plugin/maxcompute` directory within your Presto installation.
+
+### Configuration
+
+Before configuring the MaxCompute connector, please ensure you have prepared all the necessary information associated with your MaxCompute project, including project name, access
+ID, access key, and resource group name.
```

Comment:
```suggestion
ID, access key, and resource group name.

```
Suggest a blank line here to left-justify the next sentence "Follow the steps below..."